packages/spans/src/index.ts

```diff
@@ -0,0 +1,199 @@
+import { NativeModules } from 'react-native';
+import { Attributes, Events, SPAN_ERROR_CODES } from '../interfaces/ISpans';
+import {
+  convertMSToNano,
+  createFalsePromise,
+  validateAndLogRequiredProperties,
+} from './Utils';
+
+export const startSpanWithName = (
+  name: string,
+  parentSpanId?: string
```

Comment:
if you can do a default parameter to null here, that would be useful - most folks will be creating parent-less spans to start off with so not making them pass a null would be great.

or does TypeScript still do that for you if you drop trailing parameters still? JS did and I hated it so I was hoping TS would've removed that lol

Reply:
JS and TS will pass it as undefined (similar to null)